src/components/BoardUsers/BoardUser.scss

```diff
@@ -19,21 +19,20 @@
 }
 
 .rest-users {
-  color: $color-backlog-blue;
-  border: 2px solid $color-backlog-blue;
-  height: $icon--extralarge;
-  width: $icon--extralarge;
+  background-color: #333948;
+  color: $color-white;
+  height: 36px;
+  width: 36px;
 }
 
 .user__initials,
 .rest-users__count {
   position: relative;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  height: 95%;
   color: var(--accent-color);
   font-weight: bold;
+  line-height: 36px;
+  font-size: 14px;
```

Comment:
`$text-size--medium: 16px;` or `$text-size--small: 12px;`. Would you mind taking one of them?

Reply:
<img width="175" alt="Screenshot 2022-09-16 at 13 48 35" src="https://user-images.githubusercontent.com/36969812/190632312-1a356cde-c112-448b-a699-4e5fea9096b2.png">
I think 16px are fine, so I'll take that :)